_query-dsl/term/prefix.md

```diff
@@ -63,8 +63,9 @@ The `<field>` accepts the following parameters. All parameters except `value` ar
 Parameter | Data type | Description
 :--- | :--- | :---
 `value` | String | The term to search for in the field specified in `<field>`.
+`boost` | Floating-point | Boosts the query by the given multiplier. Useful for searches that contain more than one query. Values in the [0, 1) range decrease relevance, and values greater than 1 increase relevance. Default is `1`. 
```

Comment:
@imamuray all [PrefixQueryBuilder](https://github.com/opensearch-project/OpenSearch/blob/2.x/server/src/main/java/org/opensearch/index/query/PrefixQueryBuilder.java#L96-L103) supports are `value`, `rewrite` and `caseInsensitive'. Any reason for this change?
Are you trying to mention the `boost` keyword of the index mappings here?

Reply:
I missed the comment [here](https://github.com/opensearch-project/documentation-website/pull/6700#issuecomment-2005650470). Ignore my previous comment. The `boost` here is keyword field type defined [here](https://opensearch.org/docs/latest/field-types/supported-field-types/keyword/#parameters). To make it synchronize across all the term queries we should either add `boost` in all of them or remove from few of them.

If we want to add it in all of them, we can use the same description 
```
A floating-point value that specifies the weight of this field toward the relevance score. Values above 1.0 increase the field’s relevance. Values between 0.0 and 1.0 decrease the field’s relevance. Default is 1.0.
```

Reply:
@owaiskazi19 Thank you for the review.
I have added `boost` to all term-level queries. https://github.com/opensearch-project/documentation-website/pull/6700/commits/080f57c3e0c12e579b66d146210e4445c140e2b1